ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/participant/state/kvutils/committer/ConfigCommitterSpec.scala

```diff
@@ -97,36 +119,28 @@ class ConfigCommitterSpec extends WordSpec with Matchers {
         case StepContinue(_) => fail
         case StepStop(actualLogEntry) =>
           actualLogEntry.hasRecordTime shouldBe false
-          actualLogEntry.hasConfigurationEntry shouldBe true
```

Comment:
Is that checked somewhere else?

Reply:
Yes, there's a test case below `produce a log entry ...` that does this check (and more) hence I removed this line to make the test cases independent.